smart_shepherd/ml_api/app/utils/postprocessor.py

```diff
@@ -30,7 +30,7 @@ def update_prediction(self, id, ngsild_prediction):
 
 		orion = config.URL_AI_PROVIDER_ORION
 		header = config.HEADER
-		url = f"http://{orion}/ngsi-ld/v1/entities/{id}/attrs"
+		url = f"http://{orion}:1026/ngsi-ld/v1/entities/{id}/attrs"
```

Comment:
You should include the port in the configurable endpoint. The port can(and will) change.

Reply:
I thought that 1026 is the port which ORION-LD uses by default, and therefore it is mandatory to have it to establish connections. Or do you mean even if it is the case in the actual implementation(which could change), I should make it configurable?

Reply:
By default means, it can change;) In normal deployments, you will have some kind of loadbalancer in front of orion-ld, that most likely will change the port. F.e. to the http default 80 or https default 443. Or, if you run multiple locally you could have one at 1026, one at 1027 etc. You should never assume ports to be fixed.